failures-summary-and-bottle-result/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Failures summary and bottle result
+description: Output the logs of failures summary and brew bottle result
```

Comment:
How about something like this?
```suggestion
description: Output the logs of the bottle generation and a summary of all step failures
```

Reply:
Not all steps bottle something though, some run a test. Others download dependencies.

Reply:
I see, seems like I miss something there